cmd/minikube/cmd/env.go

```diff
@@ -55,6 +64,19 @@ func buildDockerEnvShellOutput(envMap map[string]string) string {
 	return output
 }
 
+func buildUnsetShellOutput() string {
+	envArr := []string{"DOCKER_TLS_VERIFY", "DOCKER_HOST", "DOCKER_CERT_PATH"}
```

Comment:
Should this be retrieved from `cluster.GetHostDockerEnv` & iterating over those keys instead? Could get out of sync otherwise.